### PR TITLE
added kubernetes legend

### LINKED ISSUE #11704: [Docs] Kubernetes Legend
### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Notes for Reviewers**

This PR fixes #11704

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7411dbc252a68f10f60662e9691c1f0fa7828db7 at: https://66c46ae3603491fbaed4f290--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ef25b84996af9340e27d8ba017cdae6a728d22a6 at: https://66c47929b27a570095b588c5--meshery-docs-preview.netlify.app

File: docs/_data/kubenetes-legend.yml
Comment (vishalvivekm) [inline docs/_data/kubenetes-legend.yml:1]: typo `kubenetes` -> `kubernetes`

File: docs/pages/guides/component-legend/kubernetes-shape-guide.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      {% endif %}
+       <div style="text-align:center;">{{ shape.Shape }}</div>
+      </div>
+      <div class="shapes-details">
+      <div>{{ shape.Description }}</div>
+      </div>
+      </div>
+
+    {% endfor %}
+ </div>
```
Comment (vishalvivekm) [inline docs/pages/guides/component-legend/kubernetes-shape-guide.md:94]:
```suggestion
{% include extension-guide.html
 data_file="kubernetes-legend"
 guide_title="Shape"
 guide_description="Description"
 guide_svg="SVG"
 guide_assests_folder="kubernetes-legend"
%}
```

**Comment by l5io** [comment]
🚀 Preview for commit 8182d108ce03cfd45409356fbc58d2a7cba6f372 at: https://66c4961fb8f7c01903c8907d--meshery-docs-preview.netlify.app

**Pull request closed without merging**